sound/soc/sof/intel/hda-stream.c

```diff
@@ -462,47 +462,30 @@ irqreturn_t hda_dsp_stream_interrupt(int irq, void *context)
 {
 	struct hdac_bus *bus = context;
 	struct sof_intel_hda_dev *sof_hda = bus_to_sof_hda(bus);
-	u32 stream_mask;
 	u32 status;
-
-	if (!pm_runtime_active(bus->dev))
-		return IRQ_NONE;
+	int ret = IRQ_WAKE_THREAD;
```

Comment:
@ranj063 Waking the thread is a big change. I had a draft addition to PR1011 that only wakes stream thread if CIS or SIS bits are set, but then one needs to use a mask. Maybe this is better, I'll give it a test.